taxonomies/ingredients.txt

```diff
@@ -8628,6 +8628,7 @@ sv:pastöriserad vispgrädde
 <en:cream
 en:whipped cream
 ca:crema de llet montada, nata montada
+da:piskefløde
```

Comment:
this is already under "en:whipping cream" too (along with german and dutch duplicates)

Reply:
Right, we should keep the distinction between whipped cream and whipping cream (which is not yet whipped :) )